.eslintrc

```diff
@@ -81,7 +81,6 @@ rules:
     - 2
     - allow-null
   no-alert: 2
-  guard-for-in: 2
```

Comment:
Removed for convenience; I'd be willing to put this back, but it creates unnecessary checks when we know the contents of the object we're being passed.